bob-api/api/de_onderwijzers_api.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace api;
+
+use ErrorException;
+
+/**
+ * Class DeOnderwijzers
+ *
+ * Wrapper for de API die de onderwijzers beschikbaar hebben gesteld.
+ *
+ * Documentatie die goed is om te lezen:
+ * - https://graphql.org/learn/queries
+ * - https://graphql.org/learn/queries/#variables
+ *
+ * Als je debug logging aan wilt hebben, zet dan $DEBUG_ENABLE naar true
+ *
+ * GraphiQl docs: http://api-test.edwh.nl/graphql/?api_key=IBFiOq8Fyoxcqxo4gyDT7pBXzC0REahqcOXu1Nql
+ *
+ * @package api
+ */
+class DeOnderwijzers
+{
+    public static $DEBUG_ENABLED = false;
+
+    /**
+     * The API key to use during the connection
+     */
+    const API_KEY = "IBFiOq8Fyoxcqxo4gyDT7pBXzC0REahqcOXu1Nql";
+
+    /**
+     * The base url of the external connection
+     */
+    const BASE_URL = "http://api-test.edwh.nl/graphql/?api_key=";
+
+    /**
+     * Session token for the application. This should only be used for non-user specific actions.
+     * Once we have a user, his/her sessionToken should be used for subsequent requests.
+     */
+    private $applicationSessionToken;
+
+    /**
+     * DeOnderwijzers constructor.
+     *
+     * Create a new session token that will be used for all subsequent requests
+     * @throws ErrorException
+     */
+    public function __construct()
+    {
+        $this->applicationSessionToken = $this->generateSessionToken();
```

Comment:
Hey Bob, misschien is het beter om een duidelijkere variable naam te gebruiken dan this?

Reply:
Als het zou kunnen zou ik het doen. `$this` is echter een keywoord vanuit PHP dus dat kan ik niet veranderen. Wel goede comment though!